torch/quantization/fx/quantize.py

```diff
@@ -374,15 +376,24 @@ def load_arg(a):
                 if qconfig is None:
                     continue
 
-                def insert_observer(node, observer, device):
+                def insert_observer_module(node, observer, device):
                     get_new_observer_name = get_new_attr_name_with_prefix(prefix)
                     observer_name = get_new_observer_name(model)
+                    if device:
+                        observer.to(device)
                     setattr(model, observer_name, observer)
                     self.activation_post_process_map[node.name] = observer
+                    return observer_name
+
+                # insert a call to activation_post_process module
+                def insert_observer_call(node, observer_name):
                     env[node.name] = observed_graph.create_node('call_module', observer_name, (load_arg(node),), {})
+                    activation_post_process_name_map[node.name] = observer_name
```

Comment:
Shouldn't this be `self.activation_post_process`?

Reply:
oh this is storing the names of activation_post_process modules, since we need this information when we insert observer calls, and it's not stored anywhere else.

Reply:
this can be a local variable since it's just needed when we process the graph